tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -349,9 +349,18 @@ public function it_should_prevent_replacing_a_webp_image() {
 			TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/balloons.webp'
 		);
 
-		$tag = wp_get_attachment_image( $attachment_id, 'medium', false, array( 'class' => "wp-image-{$attachment_id}" ) );
+		$tag          = wp_get_attachment_image( $attachment_id, 'medium', false, array( 'class' => "wp-image-{$attachment_id}" ) );
+		$expected_tag = $tag;
+		$metadata     = wp_get_attachment_metadata( $attachment_id );
+		foreach ( $metadata['sizes'] as $size => $properties ) {
+			$expected_tag = str_replace( $properties['sources']['image/webp']['file'], $properties['sources']['image/jpeg']['file'], $expected_tag );
+		}
```

Comment:
Thanks for pointing it out. 

Now we use https://github.com/WordPress/performance/blob/fix/372-discard-larger-webp/modules/images/webp-uploads/load.php#L486 for getting additional mime types. When you upload `.webp` then it will also generate a smaller version of the `.jpeg` image and display it on the frontend side.

I think we have to either remove these tests or change them.

Reply:
Ok, thanks.